script/version-utils.js

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
+
+const { logger, spawnOrFail, prompt, shouldContinuePrompt, quit, fs, path } = require('./cli-utils');
+const currentVersion = require('../package.json').version;
+
+const isPreRelease = (version) => {
+  //Check whether there is a build version at the end (e.g., 3.0.0-beta.0)
+  return version.split('.')[3] >= 0;
```

Comment:
That is true. Main reason i did it because I am not sure whether we will call it `beta` or `alpha` or `rc`. Do it this way means it does not matter since we just check for pre-release build version. So if we decide to do alpha, rc later, we don't have to change this function.

Reply:
Got it, makes sense hence the other comment on listing few examples to help read through this script :).